src/assets/icons/Icon.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from "react";
+import Apple from "./Apple";
+import Facebook from "./Facebook";
+import Logo from "./Logo";
+import Menu from "./Menu";
+import Twitter from "./Twitter";
+
+type IconProps = {
+  name: string;
+};
+
+const Icon: FC<IconProps> = ({ name }) => {
```

Comment:
I think it will be better if this `Icon` is in `components` because it's more like a component that could have some props like container, onPress, disabled, etc. in order to be reusable. So, it isn’t just an asset. WDYT

Reply:
It makes more sense for sure, thanks! I changed it to `components/common` as they are used from everywhere in the app